Migrated gpt2 examples

Gy-Lu opened the pull request.

FrankLeeeee commented:
Hi @Gy-Lu , can you provide details on how to prepare the webtext dataset? Meanwhile, please remove the NCCL environment variables as they may not be general setting for all systems.

Gy-Lu commented:
> Hi @Gy-Lu , can you provide details on how to prepare the webtext dataset? Meanwhile, please remove the NCCL environment variables as they may not be general setting for all systems.

Hi, @FrankLeeeee, I've updated the readme. Removed the NCCL environment and added how to prepare webtext dataset.
